README.md

```diff
@@ -0,0 +1,27 @@
+# Segtree
+
+This was a simulation of a tree of the segmentious variety. Of the various trees in nature, including binary, oak, and pine, the segmentatious tree from the segmiferous jungle is known to be one of the oldest living trees in existence, dating back to 6969 BC. 
+
+## Installation
+
+Use an [axe](https://github.com/dequelabs/axe-core). Save this to a directory you have access to.
+
+```bash
+chmod +x axe
+sudo su
+./axe
+```
```

Comment:
Consider adding a comment to indicate the end of this block of code. See c++ header guards.